azureml/index.py

```diff
@@ -1,3 +1,4 @@
+from rag_experiment_accelerator.checkpoint.null_checkpoint import NullCheckpoint
```

Comment:
Minor: put it along with other internal imports, please.

Reply:
Thanks, moved